tracer/src/Datadog.Tracer.Native/cor_profiler.h

```diff
@@ -61,7 +61,7 @@ class CorProfiler : public CorProfilerBase
     std::unique_ptr<TypeReference> trace_annotation_integration_type = nullptr;
     std::unique_ptr<TracerRejitPreprocessor> tracer_integration_preprocessor = nullptr;
     bool trace_annotations_enabled = false;
-
+    bool call_target_bubble_up_exception_available = true;
```

Comment:
I tend to agree, but first just verify with the Mismatch project tests that indeed you enter into this code block if there is a version mismatch just to be sure.

There is one last point that is missing: If we have multiple app domains (in .NET Framework), and each app domain loads different `Datadog.Trace.dll` assembly, then are we in a situation where the type might seem to be exist while it's actually missing (in the situation where one of the managed lib is > ver 22 and one is older)?
I am not sure though if in the automatic instrumentation we might be using the referenced `Datadog.Trace` (in case our customer is also using manual instrumentation).
/cc @zacharycmontoya know more than I do regarding this versioning hell.

Reply:
Yes I checked and the code is called for the mismatch projects, also tests weren't passing before when there was no such code.
mhh interesting for the app domains.. so I'm not sure I understand very well, in case of multiple app domains, wouldn't `EnsureCallTargetBubbleUpExceptionTypeAvailable` be called every time, for one app domain with the type rejitting with the filter, for the other app domain, rejitting without the filter?

Curious about it^^ not sure how to test it on IIS though 🤔

Reply:
ok thanks for the clarification, indeed if it sets to true for all the other loaded ones, which might not contain the type, that's not great, so I changed it in 60801430740c25055742d00fabbe5365e900fe45 , thanks for having thought about it!